src/views/LimitedRestorationExtension.vue

```diff
@@ -184,9 +205,6 @@ export default class Restoration extends Vue {
     // do not proceed if app is not ready
     if (!val) return
 
-    // do not proceed if we are not authenticated (safety check - should never happen)
-    if (!this.isAuthenticated) return
-
```

Comment:
This appears unnecessary as there's a navigation guard that performs this check.